R/pkg/R/DataFrame.R

```diff
@@ -1859,7 +1859,7 @@ setMethod("where",
 #' @param colnames A character vector of column names.
```

Comment:
can we update this to indicate that we accept the varargs style ? I think we can document `...` as a `@param` for this